<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>****************************************************************************
**
** Copyright (C) 2012-2025 Donna Whisnant, a.k.a. Dewtronics.
** Contact: http://www.dewtronics.com/
**
** This file is part of the KJVCanOpener Application as originally written
** and developed for Bethel Church, Festus, MO.
**
** GNU General Public License Usage
** This file may be used under the terms of the GNU General Public License
** version 3.0 as published by the Free Software Foundation and appearing
** in the file gpl-3.0.txt included in the packaging of this file. Please
** review the following information to ensure the GNU General Public License
** version 3.0 requirements will be met:
** http://www.gnu.org/copyleft/gpl.html.
**
** Other Usage
** Alternatively, this file may be used in accordance with the terms and
** conditions contained in a signed written agreement between you and
** Dewtronics.
**
****************************************************************************</comment>
 <class>CPassageNavigator</class>
 <widget class="QWidget" name="CPassageNavigator">
  <property name="windowTitle">
   <string>PassageNavigatorForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layoutDirectReference">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblBookDirect">
         <property name="text">
          <string>&amp;Book:</string>
         </property>
         <property name="buddy">
          <cstring>comboBookDirect</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBookDirect"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="lblChapterDirect">
         <property name="text">
          <string>C&amp;hapter:</string>
         </property>
         <property name="buddy">
          <cstring>comboChapterDirect</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboChapterDirect"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="lblVerseDirect">
         <property name="text">
          <string>&amp;Verse:</string>
         </property>
         <property name="buddy">
          <cstring>comboVerseDirect</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboVerseDirect"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="lblWordDirect">
         <property name="text">
          <string>&amp;Word:</string>
         </property>
         <property name="buddy">
          <cstring>comboWordDirect</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboWordDirect"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="lineDirectReference">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblStartRef">
         <property name="text">
          <string>&amp;From Location:</string>
         </property>
         <property name="buddy">
          <cstring>editStartRef</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editStartRef">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>201</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>201</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblVerse">
         <property name="text">
          <string>&amp;Verse:</string>
         </property>
         <property name="buddy">
          <cstring>spinVerse</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CSpinBox" name="spinVerse"/>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblChapter">
         <property name="text">
          <string>C&amp;hapter:</string>
         </property>
         <property name="buddy">
          <cstring>spinChapter</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CSpinBox" name="spinChapter"/>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblBook">
         <property name="text">
          <string>&amp;Book:</string>
         </property>
         <property name="buddy">
          <cstring>spinBook</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CSpinBox" name="spinBook"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblWord">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Word:</string>
         </property>
         <property name="buddy">
          <cstring>spinWord</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CSpinBox" name="spinWord"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblResolved">
         <property name="text">
          <string>Resolves &amp;To:</string>
         </property>
         <property name="buddy">
          <cstring>editResolved</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editResolved">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>201</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>201</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="6">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblTestament">
         <property name="text">
          <string>Bible/T&amp;estament:</string>
         </property>
         <property name="buddy">
          <cstring>comboTestament</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboTestament">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>22</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="5" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="chkboxReverse">
         <property name="text">
          <string>Reverse &amp;Direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblRefType">
         <property name="text">
          <string>T&amp;ype:</string>
         </property>
         <property name="buddy">
          <cstring>comboRefType</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboRefType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="linePassageReference">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutPassageReference">
     <item>
      <widget class="CPassageReferenceWidget" name="widgetPassageReference" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="editVersePreview">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>150</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPassageReferenceWidget</class>
   <extends>QWidget</extends>
   <header>PassageReferenceWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CSpinBox</class>
   <extends>QSpinBox</extends>
   <header>SubControls.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinWord</tabstop>
  <tabstop>spinVerse</tabstop>
  <tabstop>spinChapter</tabstop>
  <tabstop>spinBook</tabstop>
  <tabstop>comboTestament</tabstop>
  <tabstop>editStartRef</tabstop>
  <tabstop>editResolved</tabstop>
  <tabstop>editVersePreview</tabstop>
  <tabstop>comboBookDirect</tabstop>
  <tabstop>comboChapterDirect</tabstop>
  <tabstop>comboVerseDirect</tabstop>
  <tabstop>comboWordDirect</tabstop>
  <tabstop>comboRefType</tabstop>
  <tabstop>chkboxReverse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
